src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/dashboard/dashboard.component.scss

```
@@ -1,3 +1,3 @@
-main {
+.main-padding {
```

CSS class names should ideally describe components or what they are for, not what they internally do. If what you want to do here is adding some padding to the first item only for the Dashboard v2, you can simply apply the padding rule to the `cd-refresh-selector` or `cd-health`.

Or even better, use the CSS conditional pseudo-class [`:has()`](https://developer.mozilla.org/en-US/docs/Web/CSS/:has):
```suggestion
main:has(cd-refresh-selector) {
```

That said, I'd personally remove the refresh-selector (I've never seen any users using that).

But given that's becoming dead code soon... BTW, we shouldn't forget to remove the Dashboard v2... when? For S-release? @nizamial09 ? Let's file a tracker not to leave any dead code around.

---

Yup, we talked about removing the landing page in the next release which is S. I'll create tracker for it.

---

https://tracker.ceph.com/issues/58961